Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,163 @@
+//
+//  CDTFetchChanges.h
+//  
+//
+//  Created by Michael Rhodes on 31/03/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@class CDTDatastore;
+@class CDTDocumentRevision;
+
+/**
+ Read changes to a database.
+ 
+ CDTFetchChanges reads the changes to a database from a given sequence value. It provides
+ callbacks for updated and created documents (documentChangedBlock) and deleted documents
+ (documentWithIDWasDeletedBlock). It also provides a completion block for when all changes
+ have been received and processed.
+ 
+ The CDTFetchChanges object will always return _all_ changes from the given sequence value.
+ 
```

Comment:
Saying that it returns "_all_" changes may be confusing since this class filters out non-winning revisions. I think it might be especially confusing to a couchdb/cloudant veteran. 

I think your point is that each time you call ```-start``` with a particular value for ```previousServerSequenceValue```, the winning revision for all documents with changes since ```previousServerSequenceValue``` will be supplied to the appropriate callback block.